README.md

```diff
@@ -191,6 +191,25 @@ batch = TransitionMiniBatch(transitions, n_frames=4)
 batch.observations.shape == (32, 4, 84, 84)
 ```
 
+## Docker
```

Comment:
For now, I consider that this instruction should be placed at the documentation since README has many information.